src/main/java/seedu/duke/reflection/ReflectionQuestion.java

```diff
@@ -0,0 +1,35 @@
+package seedu.duke.reflection;
+
+/**
+ * ReflectQuestion class is used to create reflect question objects.<br/>
+ * <br/>
+ * The static attribute questions is used to keep track all the questions created. <br/>
+ * Attribute isLike is to keep track of whether a user likes this question or not.
```

Comment:
Personal opinion is to describe the attributes as a comment directly above the attribute's declaration, not in the class-level JavaDoc. These attributes are private after all; The actual variable name should be kept abstract and not revealed.